doc/nrf/protocols/wifi/advanced_modes/wifi_advanced_security_modes.rst

```diff
@@ -0,0 +1,52 @@
+.. _ug_nrf70_wifi_advanced_security_modes:
+
+Wi-Fi Enterprise test: X.509 Certificate header generation
```

Comment:
Seems like the text above this section is removed. Is it intentional?

Reply:
no, it was a gap, I fixed it now and also reworked the doc location and added an intro para.